index.php

```diff
@@ -127,6 +127,50 @@
                 <a class="read-more" href="https://usn.ubuntu.com/releases/ubuntu-18.04-lts/">Security Notices</a>
             </div>
         </section>
+        <section id="workflow" class="grey">
+            <div class="grid">
+                <div class="two-thirds">
+                    <h2>Get work done. Or play.</h2>
+                    <h4>elementary OS is designed to keep you productive and focused by getting out of your way with Multitasking View, Picture-in-Picture, Do Not Disturb, and more. And by staying out of your way, elementary OS is great when you're watching videos or playing games, too.</h4>
```

Comment:
I feel like mentioning "elementary OS" each time is a bit repetitive since this whole page is in the context of elementary OS. Maybe we could do something like:

```suggestion
                    <h4>Stay productive and focused with Multitasking View, Picture-in-Picture, Do Not Disturb, and more. Or keep work out of sight when watching videos or playing games.</h4>
```

Or in general just something more concise. Otherwise each section is saying "elementary OS is designed to" at the beginning of each paragraph